index.md

```diff
@@ -65,7 +65,7 @@ Many software programs already use `--color` or `-c` to enable color output. Thi
 | Software | Description | Date / Version Supported |
 |-|-|-|
 | [node](https://nodejs.org/node) | JavaScript runtime | |
+| [Nox](https://github.com/wntrblm/nox) | Flexible test automation | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
```

Comment:
Thoughts or objections on simply adding `for Python` - I think it's helpful to include for people what language things are for incase they are looking for something they can use.

```suggestion
| [Nox](https://github.com/wntrblm/nox) | Flexible test automation for Python | [2022-01-07 / 2022.1.7](https://github.com/wntrblm/nox/releases/tag/2022.1.7) |
```